### ISSUE TITLE
Installation fails due to conflicting base58 version

### ISSUE THREAD

**Issue by NeolithEra** [jansenmarc/WavesGatewayFramework#26]
Hi, users are unable to run **_WavesGatewayFramework_** due to dependency conflict with _**base58**_ package. 
As shown in the following full dependency graph of **_WavesGatewayFramework_**, **_WavesGatewayFramework_** requires _**base58>=0.2.5**_，while **_PyWaves_** requires _**base58==0.2.5**_.

According to pip’s _“first found wins”_ installation strategy, _**base58 1.0.3**_ is the actually installed version. However, _**base58 1.0.3**_ does not satisfy _**base58==0.2.5**_.

### Dependency tree
```
WavesGatewayFramework-master
| +-base58(version range:>=0.2.5)
| +-flask(version range:>=0.12.2)
| | +-click(version range:>=5.1)
| | +-itsdangerous(version range:>=0.24)
| | +-jinja2(version range:>=2.10.1)
| | | +-markupsafe(version range:>=0.23)
| | +-werkzeug(version range:>=0.15)
| +-gevent(version range:>=1.2.2)
| +-pymongo(version range:>=3.4.0)
| +-python-doc-inherit(version range:>=0.3.0)
| +-pywaves(version range:>=0.8.8)
| | +-base58(version range:==0.2.5)
| | +-pyblake2(version range:*)
| | +-python-axolotl-curve25519(version range:*)
| | +-requests(version range:*)
| | | +-certifi(version range:>=2017.4.17)
| | | +-chardet(version range:<3.1.0,>=3.0.2)
| | | +-idna(version range:>=2.5,<2.9)
| | | +-urllib3(version range:<1.26,>=1.21.1)
| +-requests(version range:>=2.9.1)
| | +-certifi(version range:>=2017.4.17)
| | +-chardet(version range:<3.1.0,>=3.0.2)
| | +-idna(version range:>=2.5,<2.9)
| | +-urllib3(version range:<1.26,>=1.21.1)
| +-simplejson(version range:>=3.11.1)
```

Thanks for your help.
Best,
Neolith

**Comment by NeolithEra** [comment]
### Solution

1.	Fix your direct dependency to be base58==0.2.5. I have checked this revision will not affect your downstream projects now.

2.	Ask your upstream project PyWaves for help to loosen the version range of base58.
Which solution do you prefer, 1 or 2?
@henning96 Please let me know your choice. I can submit a PR to solve this issue.

**Comment by PyWaves** [comment]
I would prefer solution 1).

**Comment by henning96** [comment]
See: https://github.com/jansenmarc/WavesGatewayFramework/pull/27#issuecomment-518744841

**Comment by henning96** [comment]
Fixed in Release 1.0.5
